site/content/en/docs/Guides/access-api.md

```diff
@@ -108,6 +117,18 @@ func main() {
 	fmt.Printf("New game servers' name is: %s", newGS.ObjectMeta.Name)
 }
 ```
+{{% feature publishVersion="1.2.0" %}}
```

Comment:
Same here, not sure if we need the feature shortcode here. It's not a new feature, just a new example. WDYT?

Reply:
I will change {{< release-branch >}} in lines below to a master link then